src/devana/code_generation/printers/default/functionprinter.py

```diff
@@ -78,6 +78,8 @@ def print(self, source: FunctionInfo, config: Optional[PrinterConfiguration] = N
             result = "inline " + result
         if source.modification.is_constexpr:
             result = "constexpr " + result
+        if source.modification.is_consteval:
```

Comment:
Please also add printer to constructor (yes, constructor has dedicated printer in class printers file).